aria-practices.html

```diff
@@ -53,23 +53,39 @@
+      Accessibility semantics are the meaning of user interface elements that need to be conveyed to assistive technology users in order for those users to understand and use the elements.
+      For example, people who can visually perceive a search icon button understand the element can be activated to perform a search by the way it is styled and positioned.
+      To make that icon accessible to a screen reader user, one of the semantics that needs to be communicated is that the element represents a button, and there are several ways that semantic can be conveyed to assistive technologies in code.
+      In addition, for the button to be accessible via a keyboard, it needs to be keyboard focusable, and it is commonly expected that when focused, pressing <kbd>Enter</kbd> or <kbd>Space</kbd> will activate it.
+      The APG describes how to convey accessibility semantics and implement keyboard accessibility for many common design patterns, ranging from typical buttons and popup menus to complex tree grids.
+      It also explains essential practices related to these patterns, such as how to convey web page structure with ARIA landmark regions and effectively utilize the many ways of encoding accessible names.
+    </p>
+    <p>
+      The APG is organized into two major sections: patterns and practices.
+      Each pattern explains how to make a common user interface element, such as a button, menu, or dialog, accessible, and provides functional example implementations of the pattern.
+      The practices section gives in-depth explanation of how to satisfy a variety of accessibility needs that surface when making rich internet application experiences accessible.
+      For instance, the practices section on providing accessible names and descriptions gives detailed description of seven different naming techniques as well as a table providing guidance for naming more than 80 types of elements.
+    </p>
+    <p>
+      The APG assumes basic understanding of web development, especially of HTML and CSS.
+      It may be particularly useful to designers and engineers who are working on site design systems and component libraries.
+      However, it is important to understand that the APG covers only a portion of what is needed to create accessible experiences.
+      To understand accessibility more broadly, it is advisable to start with the <a href="https://www.w3.org/WAI/fundamentals/">WAI Accessibility Fundamentals</a> and <a href="https://www.w3.org/WAI/design-develop/">WAI Overview of Design and Development</a>.
+    </p>
+    <p>
+      The accessibility semantics used in the APG patterns and described in APG practices are defined in the
+      <cite><a href="https://www.w3.org/TR/wai-aria-1.2/">WAI-ARIA 1.2 Specification</a></cite>.
+      That is, the ARIA specification standardizes the meaning of the attributes that describe elements and their states to assistive technologies, e.g., the meaning of <code>role="button"</code>.
```

Comment:
Change "e.g., the..." to "such as the" (less punctuation clutter, since there's lots of commas around there already)